fix.txt

```diff
@@ -0,0 +1,12 @@
+# FAILED unit/examples/test_notebooks_python.py::test_sar_deep_dive_runs - papermill.exceptions.PapermillExecutionError:  
```

Comment:
Another thing, since we joined the LF, there is a new action we need to pass: DCO. You noticed that it failed. The solution here: https://github.com/recommenders-team/recommenders/wiki/How-to-sign-commits